.github/workflows/publish-website.yml

```diff
@@ -0,0 +1,31 @@
+# With thanks to https://bluegenes.github.io/mkdocs-github-actions/
+name: Build and publish shakenfist.com
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: "recursive" 
```

Comment:
Do we need submodules? But otherwise looks good

Reply:
We do not. I shall remove that.